### ISSUE TITLE
Accessibility for screen readers when comparing files in vscode

### ISSUE THREAD

**Issue by jeanbrazcosta** [microsoft/vscode#152511]
Currently I've been trying to use the file comparer in vscode or even git extension, but the problem is that, when getting the two editors with the original file and with the modified file, there is no easy way for those who use screen reader to know the that has been modified.

My suggestion is to have some key combination that takes the cursor to the line that was modified and in the editor that has the original file, the cursor is also taken to the line as it was before the change.

Assuming I have changed line 3 of a file so when using a combination of keys after having opened the compare option, curosr would be taken to line 3, when pressing shift tab to see the original file the cursor would already be on line 3 and so on.

This would make the code review a lot easier because I believe that for those who don't use a screen reader, the changes are colored.

This feature is possible in eclipse, but I couldn't do it using vscode.

Thanks

**Comment by jeanbrazcosta** [comment]
Good holidays, enjoy a lot, thanks the return.
Enviado do meu iPhone

**Comment by isidorn** [comment]
@jeanbrazcosta can you please try the `F7` / `Shift + F7` experience. This one is designed to be accessible and we would be very interested to hear if you hit any limitations for this?
Also we should consider how to make it more discoverable. Thanks a lot and sorry for the slow response.

**Comment by jeanbrazcosta** [comment]
@isidorn The feature turned out great!

The only thing I couldn't see changes after pressing f7, was when I made only formatting change in my code.

To reproduce, I opened an original file and on any line I pressed several enter to get line breaks and this comparison did not appear in the resource

If more, it was excellent

Thanks!

**Comment by isidorn** [comment]
@jeanbrazcosta thanks for the feedback. If you change the following setting might it work then
`"diffEditor.ignoreTrimWhitespace": false`

Because I just tried adding 3 new lines, and pressing F7 in the diff view and the screen reader properly read to me "blank modified line".
So if with the setting it still does not work do let us know what exact steps you are doing and what behaviour you would like to differ.

Also any other feedback regarding the F7 experience is very welcome! Thanks!

**Comment by jeanbrazcosta** [comment]
@isidorn, actually my settings were ignoring whitespace, I made the change you mentioned and the reader was able to report the formatting changes as well.

In my opinion, the feature turned out great, it was what I was expecting.

Thanks!

**Comment by isidorn** [comment]
Glad to hear you like the feature. If you have any additional feedback do let us know.
As follow up to make it more discoverable I have created this one https://github.com/microsoft/vscode/issues/158929
Thanks
